org.godotengine.godot.BaseApp.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: org.godotengine.godot.BaseApp
+branch: '3.1'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+
+build-options:
+  arch:
+    x86_64:
+      env:
+        # Only enable link-time optimization when targeting x86_64
+        # (causes issues on other architectures)
+        SCONS_FLAGS_EXTRA: use_lto=yes
+
+  env:
+    # Will be appended to the version string displayed in the command-line help
+    BUILD_NAME: flathub
+
+    # SCons flags common to all builds
+    # Shouldn't be quoted when used as it's a single string, not an array
+    SCONS_FLAGS: >
+      platform=x11
+      CCFLAGS=-I/app/include
+      prefix=/app
+      unix_global_settings_path=/app
+      progress=no
+      builtin_freetype=no
+      builtin_libogg=no
+      builtin_libpng=no
+      builtin_libtheora=no
+      builtin_libvorbis=no
+      builtin_libwebp=no
+      builtin_openssl=no
+      builtin_libvpx=no
+      builtin_zlib=no
+      udev=no
+
+finish-args:
```

Comment:
Do these get inherited by apps?

Reply:
You're correct, some of them can be removed here. Not all projects require network access.

Or should I just remove all of them and let derived applications define `finish-args` themselves?

Reply:
I ended up removing all `finish-args`. I've seen this in some other BaseApps, so I figure letting depending applications define them manually is the best solution.